samples/java/getting-started/webfrontend/src/main/resources/static/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE HTML>
```

Comment:
What do you think about using the same static content as the Node.js app so there is a little more substance to the webfrontend (it uses Angular in a very minimal fashion).